torch/csrc/jit/script/init.h

```diff
@@ -6,6 +6,11 @@ namespace torch {
 namespace jit {
 namespace script {
 void initJitScriptBindings(PyObject* module);
+std::shared_ptr<Graph> createGraphByTracing(
+        py::function func,
+        std::vector<tracer::TraceInput> inputs,
+        size_t num_inputs);
```

Comment:
Can we move this to `tracer.cpp`? `init.h` shouldn't really have functions like this one